src/commands/destroy.yml

```diff
@@ -97,11 +95,11 @@ steps:
 
         rm -rf .terraform
 
-        terraform init -input=false -lock-timeout=300s -no-color $INIT_ARGS "$module_path"
+        terraform -chdir="$module_path" init -input=false -lock-timeout=300s -no-color $INIT_ARGS
```

Comment:
This might cause issues when users run the `terraform/install` command and specify a `terraform_version` value lower than 0.14. We should have a check, otherwise the command will fail for those users.

Reply:
You are correct about the backwards compatibility but I'm unsure about the best way to handle it though:
- checking the "terraform_version" parameter in terraform/install is fairly easy but is it enough? In some cases (like mine), terraform is already installed in the environment to accelerate CI so I think the orb should check the "terraform version" output instead?
- I had to change multiple lines to use the chdir option (var-file path are also modified), so to maintain compatibility would mean to put conditions wrapping those lines, losing a lot of maintainability. How do you want to handle this? I think some additional deprecations from 0.15 might even still be missing from my PR. I didn't run it in a 0.15 build for now.

Reply:
Maybe we could split the PR into 2 pieces? One related to having parameterized plan files, and the other one related to module path refactoring.

Reply:
I did split the plan file in another PR. The way the plan_file is handled is different, so it will be difficult to keep both scripts in the same orb version I think?

As passing directory path instead of the plan_file is some legacy now that Terraform 0.15 and 1.X are GA, should it be handled by some kind of legacy orb version?